setup-machine.sh

```diff
@@ -99,11 +94,10 @@ sudo rm -rf /vx/services/* /vx/ui/* /vx/admin/*
 
 # Let vx-admin read logs
 sudo usermod -aG adm vx-admin
+sudo usermod -aG adm vx-ui
```

Comment:
@benadida Does this add permissions other than just viewing the logs? Without this even with granting kiosk-browser read permissions to /var/log I would get a permissions denied error when trying to export the logs from the app.

Reply:
yeah, I believe that is the key permission change, so I'm good with this.